test/inductor/test_torchinductor_strided_blocks.py

```diff
@@ -859,7 +824,7 @@ class TritonBlockPointerTestCPU(InductorTestCase):
 
 @unittest.skipIf(not HAS_GPU, "requires triton GPU backend")
 @config.patch("triton.use_block_ptr", True)
-class TritonBlockPointerTestGPU(InductorTestCase):
+class TritonBlockPointerTestGPU(InductorTestCase, TestUtils):
```

Comment:
TODO: use a chain from InductorTestCase -> TestUtils -> TritonBlockPointerTest as opposed to multiple inheritance?

Reply:
[fbf346e](https://github.com/pytorch/pytorch/pull/137243/commits/fbf346e82899e76d5ecdc23a116eaa78d402e0cc)